mapper/mapper.go

```diff
@@ -195,9 +195,17 @@ func CreateGetCoverage(req *http.Request, basePage coreModel.Page, lang, filterI
 
 	var results []model.SearchResult
 	for _, area := range areas.Areas {
+		var isSelected bool
+		for _, added := range p.AreasAdded {
```

Comment:
The intention is to `GET` the areas added via an endpoint, using the page model props is temporary and to facilitate local design testing. Unit tests will be updated when the endpoints are used